src/utils/query.ts

```diff
@@ -0,0 +1,49 @@
+import { IDeskproClient, useDeskproAppClient } from "@deskpro/app-sdk";
+import {
+  QueryClient,
+  QueryKey,
+  useQuery,
+  UseQueryOptions,
+  UseQueryResult,
+} from "@tanstack/react-query";
+
+export const query = new QueryClient({
+  defaultOptions: {
+    queries: {
+      suspense: true,
+      refetchOnWindowFocus: false,
+    },
+  },
+});
+
+export function useQueryWithClient<
```

Comment:
Do you think we should just commit to using React Query going forward? If so, we should add this client injection wrapper to the SDK

Reply:
Absolutely. Especially the tanstack variant that supports mutations.

Reply:
I'll do it after I get the MVP for xero out if that's ok